ts/src/base/types.ts

```diff
@@ -189,6 +189,35 @@ export interface DepositAddressResponse {
     tag?: string;
 }
 
+export interface Position {
```

Comment:
@ttodua is anything missing here?

Reply:
I think there might be : 
- marginType
- contractSize (idk about this, but some exchanges have it implemented in parser return)

Reply:
@ttodua we have already the unified `marginMode`, regarding the `contractSize` I added it because some exchanges are adding it to he response as well

Reply:
aah sorry my quick-filter snippet misleaded me, it showed that there were `marginType` (not `marginMode`) being returned from some exchanges.